intro_to_rl.ipynb

```diff
@@ -0,0 +1,3249 @@
+{
```

Comment:
<p>ctrl+f "choses" and change to "chooses" :)  </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/deep-learning-indaba/indaba-pracs-2022/pull/6/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='12'/>

Reply:
<p>haha done.</p>